spec/requests/api/v2/access_request_spec.rb

```diff
@@ -24,6 +24,79 @@
 
   subject { response }
 
+  describe 'GET #index' do
+    let(:access_requests_index_route) do
+      get "/api/v2/access_requests",
+          headers: { 'HTTP_AUTHORIZATION' => credentials }
+    end
+
+    context 'when unauthenticated' do
+      before do
+        access_requests_index_route
+      end
+
+      let(:payload) { { email: 'foo@bar' } }
+
+      it { should have_http_status(:unauthorized) }
+    end
+
+    context 'when unauthorized' do
+      let(:unauthorised_user) { create(:user) }
+      let(:payload) { { email: unauthorised_user.email } }
+      let(:unauthorised_user_route) do
+        get "/api/v2/access_requests",
+            headers: { 'HTTP_AUTHORIZATION' => credentials }
+      end
+
+
+      it "should raise an error" do
+        expect { unauthorised_user_route }.to raise_error Pundit::NotAuthorizedError
+      end
+    end
+
+    context 'when authorised' do
+      let!(:first_access_request) { create(:access_request, request_date_utc: '2019-05-05 00:10:47 UTC') }
+      let!(:second_access_request) { create(:access_request, request_date_utc: '2019-05-05 00:10:48 UTC') }
+
+      before do
+        access_requests_index_route
+      end
+
+      it 'JSON displays the correct attributes' do
+        json_response = JSON.parse response.body
+
+        expect(json_response).to eq(
+          [
+            {
+              "id" => first_access_request.id,
+              "email_address" => first_access_request.recipient.email,
+              "first_name" => first_access_request.recipient.first_name,
+              "last_name" => first_access_request.recipient.last_name,
+              "requester_email" => first_access_request.requester.email,
+              "requester_id" => first_access_request.requester.id,
+              "organisation" => first_access_request.organisation,
+              "reason" => first_access_request.reason,
+              "request_date_utc" => '2019-05-05T00:10:47.000Z',
+              "status" => first_access_request.status
+            },
+            {
+              "id" => second_access_request.id,
+              "email_address" => second_access_request.recipient.email,
+              "first_name" => second_access_request.recipient.first_name,
+              "last_name" => second_access_request.recipient.last_name,
+              "requester_email" => second_access_request.requester.email,
+              "requester_id" => second_access_request.requester.id,
+              "organisation" => second_access_request.organisation,
+              "reason" => second_access_request.reason,
+              "request_date_utc" => '2019-05-05T00:10:48.000Z',
+              "status" => second_access_request.status
+            }
+          ]
+       )
```

Comment:
Is there a reason this is not a JSON API response? I would expect it to be, offhand ...